src/Components/Form/FormFields/Autocomplete.tsx

```diff
@@ -24,6 +25,7 @@ type AutocompleteFormFieldProps<T, V> = FormFieldBaseProps<V> & {
   optionValue?: OptionCallback<T, V>;
   optionDescription?: OptionCallback<T, string>;
   optionIcon?: OptionCallback<T, React.ReactNode>;
+  optionProfilePicture?: any;
```

Comment:
Also, this is a reusable generic component. `optionProfilePicture` is too specific for a reusable generic component.

Reply:
Updated